xml_converter/src/attribute/unique_id.hpp

```diff
@@ -6,8 +6,10 @@
 #include "../rapid_helpers.hpp"
 #include "../rapidxml-1.13/rapidxml.hpp"
 
+using namespace std;
 class UniqueId {
-
+ public:
+    string guid;
```

Comment:
This will eventually need to be a byte array which is parsed from the string. Might as well do that now instead of when we start writing the protobuf files.

Reply:
Code has been changed so that guid is now a vector of bytes